persons/services/constants.py

```diff
@@ -0,0 +1,17 @@
+PIDs = {
+    "date_of_birth": "P569",
+    "birthplace": "P19",
+    "occupation": "P106",
+    "position": "P39",
+    "party": "P102",
+    "education": "P69",
+    "resident": "P551",
+    "degree": "P512",
+    "ideology": "P1142",
+}
+
+URLs = {
+    "url1": "https://www.wikidata.org/wiki/Special:EntityData/{search}.json",
+    "url2": "https://www.wikidata.org/w/api.php?action=wbsearchentities"
+    "&search={search}&language=en&format=json",
+}
```

Comment:
```suggestion
ENITITY_SEARCH_URL = "https://www.wikidata.org/wiki/Special:EntityData/{search}.json"
TERM_SEARCH_URL = "https://www.wikidata.org/w/api.php?action=wbsearchentities"
    "&search={search}&language=en&format=json"
```
This makes it more verbose